crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -26,7 +26,8 @@ def initialize(backup)
       end
 
       def restore
-        [:restore_crowbar, :run_installer, :restore_chef, :restore_database].each do |component|
+        actions = [:restore_crowbar, :run_installer, :restore_chef_keys, :restore_chef, :restore_database]
```

Comment:
Metrics/LineLength: Line is too long. [106/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)